chalice/app.py

```diff
@@ -317,12 +349,30 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
 
+class APIGateway(object):
```

Comment:
Please document this in the `docs/source/api.rst`.  We should at least tell people what the default list of binary types is.